src/lib/kdf/xmd/info.txt

```diff
@@ -0,0 +1,11 @@
+<defines>
+XMD -> 20240404
+</defines>
+
+<module_info>
+name -> "XMD"
```

Comment:
Perhaps add a `brief -> ""` here? That string shows up in the module overview in doxygen.

Reply:
Done